contracts/NFT.solpp

```diff
@@ -1,5 +1,4 @@
 // SPDX-License-Identifier: MIT
-
 pragma soliditypp ^0.8.0;
 
 import "./openzeppelin/token/ERC721/extensions/ERC721Enumerable.sol";
```

Comment:
Can recommend the contract wizard from Oz:
https://wizard.openzeppelin.com/#erc721

Reply:
👍 The ERC721/IERC721 use in our contracts can be simplified significantly. For now, I just wanted to use something to create NFTs for the tests.

Reply:
Also, I don't like uploading the contracts we import to the project and would prefer to leverage npm install them but took this approach to resolve some of the setup issues.